clients/fides-js/src/fides.ts

```diff
@@ -86,8 +90,23 @@ let _Fides: Fides;
 const updateCookie = async (
   oldCookie: FidesCookie,
   experience: PrivacyExperience,
-  debug?: boolean
+  debug?: boolean,
+  isExperienceClientSideFetched?: boolean
 ): Promise<{ cookie: FidesCookie; experience: PrivacyExperience }> => {
+  if (!isExperienceClientSideFetched) {
+    // If it's not client side fetched, we don't update anything since the cookie has already
+    // been updated earlier.
+    return { cookie: oldCookie, experience };
+  }
+  if (oldCookie.consent) {
```

Comment:
this will always be true I think— `{}` is truthy

Reply:
good catch! I've added a helper fn to help determine whether we have consent set on this obj